site/style.css

```
@@ -1,6 +1,6 @@
 :root {
-  --font-heading: 'Kalam', cursive;
-  --font-body: 'Patrick Hand', cursive;
+  --font-heading: 'Kalam', 'Inter', system-ui, sans-serif;
+  --font-body: 'Inter', system-ui, -apple-system, 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif;
```

Stylelint (17.9.0)

[error] Expected "Roboto" to be "roboto" (value-keyword-case)

(value-keyword-case)